docs/uvbeam_tutorial.rst

```diff
@@ -193,14 +193,18 @@ c) Reading in the MWA full embedded element beam
 
 d) Writing a regularly gridded beam FITS file
 **********************************************
+When reading a beam FITS file, you also have the option of selecting frequencies and
+az/za values at the read step -- i.e. so that memory is never allocated for data outside
+these ranges. Use the ``freq_range`` and ``za_range`` parameters to achieve this.
```

Comment:
maybe also list ``az_range`` here? And I think the ``freq_range`` keyword will also work for HEALPix beam fits files, right?

Reply:
Yes, added. Thank you.